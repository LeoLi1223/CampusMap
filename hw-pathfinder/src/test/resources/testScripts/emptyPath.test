# tests that a path from a node to itself

CreateGraph graph1

AddNode graph1 A
AddNode graph1 B

AddEdge graph1 A B 5.0

FindPath graph1 A A