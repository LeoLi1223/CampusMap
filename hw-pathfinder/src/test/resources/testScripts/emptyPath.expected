# tests that a path from a node to itself

created graph graph1

added node A to graph1
added node B to graph1

added edge 5.000 from A to B in graph1

path from A to A:
total cost: 0.000